### PR TITLE
1.0.3 About Release Edits

### PR DESCRIPTION
**Pull request by andyjohnson**
This is Part 1: About redone with a variety of changes.  Please see the commits for details.  As there are no requirements in this part, there are no "breaking" changes to worry about. Rather, focus on keeping terminology as consistent as possible and delivering a clear message to those reading the spec.

### CODE REVIEW

File: xAPI-About.md [hunk 1]
```diff
@@ -525,6 +570,7 @@ Clarifications and additional examples including:
 ###### 1.0.2 to 1.0.3 (TBC, 2016)
 - Complete reorganization of the document into three parts
 - Many, many clarifications
+- Definitions slightly modified and greatly clarified
```
Comment (garemoko) [inline xAPI-About.md]: Seems confusing to say they were only slightly modified but greatly clarified. Why not keep it simple and say "Revised some definitions."

**Comment by garemoko** [comment]
It looks like this is just very minor tweaks plus whitespace changes. Is that right?

In that case aside from my comment above, +1

**Comment by garemoko** [comment]
Images look fine, except that i think the LRC arrow should go in the other direction.

**Comment by garemoko** [comment]
"Learning Record Producers" appears in the intro to 2.0. Should be "Providers".

**Comment by garemoko** [comment]
"trusted exchange of information between the Learning Record Store and trusted services" should be "trusted clients"

File: xAPI-About.md [hunk 2]
```diff
@@ -312,157 +333,181 @@ IRIs always include a scheme. This is not a requirement of this standard, but pa
...
+Figure 1 shows the tracking of learning experiences. A learner has a Learning Experience. This experience could take place in an 
+online course, it could be on the job or it could be part of recreation. It really could be anything. This experience is tracked, 
+on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider may also be responsible for the 
+trusted relationship between the experience and the learner, and perhaps even launching “content” for the learner and digital 
+rights associated with the content…but it doesn’t have to be.
```
Comment (garemoko) [inline xAPI-About.md]: I'm not keen on the use of ... Can we reword this without losing the general meaning?

File: xAPI-About.md [hunk 3]
```diff
@@ -312,157 +333,181 @@ IRIs always include a scheme. This is not a requirement of this standard, but pa
...
+Figure 1 shows the tracking of learning experiences. A learner has a Learning Experience. This experience could take place in an 
+online course, it could be on the job or it could be part of recreation. It really could be anything. This experience is tracked, 
+on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider may also be responsible for the 
+trusted relationship between the experience and the learner, and perhaps even launching “content” for the learner and digital 
+rights associated with the content…but it doesn’t have to be.  
+
+The Learning Record Provider creates Learning Records and sends them to one or more Learning Record Stores (LRS). The Learner has 
+an account on the LRS and can manage access to their own records. The Learning Record Store has authentication and authorization
```
Comment (garemoko) [inline xAPI-About.md]: "The Learner has an account on the LRS and can manage access to their own records." - this isn't true for a lot of LRSs. I vote to delete.

File: xAPI-About.md [hunk 4]
```diff
@@ -312,157 +333,181 @@ IRIs always include a scheme. This is not a requirement of this standard, but pa
...
-metadata properties that may be populated. This is done through the  Activity Definition object. The Id of each 
-Activity is an IRI which also could have metadata located at where it resolves to. Any metadata where the IRI resolves is under the control of the Metadata Provider. Any metadata at the IRI is the authoritative source of metadata, and could be used to populate the LRS's canonical version of the Activity's metadata (Activity 
-Definition) as a preference to what it receives from Statements. A Metadata Consumer can access Metadata directly from the authority or can query the Activities Resource.  
+Understanding how a single Activity (as uniquely identified by its IRI) is defined and described is a key concept in xAPI. 
+Figure 2 shows this process. An Activity (as a part of a Statement) has metadata properties that may be populated within the 
+Statement itself. This is done in the Statement's Activity Definition. The Id of each Activity is an IRI which also could have 
+metadata located at where it resolves to. Any metadata where the IRI resolves is under the control of the Metadata Provider.
```
Comment (garemoko) [inline xAPI-About.md]: "located at where it resolves to" - can we reword?

**Comment by garemoko** [comment]
I think we can say this PR fixes https://github.com/adlnet/xAPI-Spec/issues/631 too

File: xAPI-About.md (hunk 3 above)
Comment (andyjohnson) [inline xAPI-About.md]: I'm fine with that.  The narrative of "controlling your own data" doesn't really show up anywhere else.  Products can sell that feature as they wish.

**Comment by andyjohnson** [comment]
Thanks so much for those, @garemoko - I got them all in the latest commit.  No disagreements on any of the comments.

**Comment by garemoko** [comment]
Thanks @andyjohnson - the only other thing was the direction of the LRC arrow in that first diagram.

**Comment by andyjohnson** [comment]
Yep, on that.  Just have to get the file overwritten.  Should just be a
minute :)

On Thu, Feb 25, 2016 at 10:03 AM, Andrew Downes <notifications@github.com>
wrote:

> Thanks @andyjohnson <https://github.com/andyjohnson> - the only other
> thing was the direction of the LRC arrow in that first diagram.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adlnet/xAPI-Spec/pull/870#issuecomment-188825469>.
>



-- 
Andy Johnson
ADL Technical Team
608-318-0049

**Comment by garemoko** [comment]
Cool. +1 once that's done.

**Comment by ljwolford** [comment]
+1

**Pull request merged**